Source/monster.h

```diff
@@ -276,7 +276,7 @@ void PrepareUniqueMonst(Monster &monster, int uniqindex, int miniontype, int bos
 void InitLevelMonsters();
 void GetLevelMTypes();
 void InitMonsterGFX(int monst);
-void monster_some_crypt();
+void WeakenNaKrul();

```

Comment:
```suggestion
void AwakenNaKrul();
```

Reply:
I never played Hellfire but I think this suppose to enable NaKrul instead of reducing his power, right?

Reply:
No. He is locked in a room. If you just open the door he will be a full strength but if you trigger spell books out side his room first he will be weakened making the fight much easier.